examples/filesystem/Makefile

```diff
@@ -32,4 +32,8 @@ USEMODULE += constfs
 # Enable to automatically format if mount fails
 #USEMODULE += vfs_auto_format
 
+# For LittleFS on real devices, the default stack size has to be
+# increased:
+CFLAGS += -DTHREAD_STACKSIZE_DEFAULT=2048
```

Comment:
Ah right it’s actually enough to bump the main thread stack since that’s the only one doing file operations. (Ok, it’s probably the only thread in this example, but still)

Reply:
It looks like this fixed it. Nice :)